In [3]:
import pandas as pd
import numpy as np 
import seaborn as sns
import matplotlib.pyplot as plt

In [4]:
df = pd.read_csv("spatial_awareness.csv")

df.head()

,Participant,Status
0,1,incorrect
1,2,correct
2,3,correct
3,4,incorrect
4,5,incorrect


In [5]:
df["Status"].unique()

array(['incorrect', 'correct', 'no_answer'], dtype=object)

In [6]:
total = df["Status"].count()
correct = (df["Status"] == "correct").sum()
incorrect = (df["Status"] == "incorrect").sum()
no_answer = (df["Status"] == "no_answer").sum()

In [7]:
print("Number of Correct awareness: ", correct)
print("Number of incorrect or not answered: ", incorrect)
print("No Data: ", no_answer)

Number of Correct awareness:  9
Number of incorrect or not answered:  7
No Data:  4


In [8]:
print(f"Number of Correct awareness: {(correct/total)*100}%")
print(f"Number of incorrect or not answered: {(incorrect/total)*100}%")
print(f"No Data: {(no_answer/total)*100}%")

Number of Correct awareness: 45.0%
Number of incorrect or not answered: 35.0%
No Data: 20.0%


In [9]:
accuracy = correct/(correct+incorrect)
print(f"Accuracy: {accuracy*100}%")

Accuracy: 56.25%


Participants demonstrate 56.25% spatial awareness accuracy rate. 

*dapat pala per session tinatanong*

Spatial awareness was measured once at the during the whole simulation rather than once during every phase. As a result, statistical comparisons between phases could not be conducted. The analysis focuses on overall descriptive performances rather than modality-specific differences. The spatial awareness were typically asked during rides 2 through 4. 

The accuracy is 56.25%. This tells us that slightly more than half of the participants are aware of their surroundings. The error rate 7 out of 20 participants. This tells us that 35% of the participants are not aware of their surroundings. 